### DISCUSSION TITLE
Third Secret in That's my Ex Calibur.

### DISCUSSION THREAD

**Discussion by Battojutsu** [MissLav/LibreQuake#249] [Q&A] [answered]
I cannot find a third secret in that's my Ex Calibur, there are supposed to be three. Could someone give me a hint?

Spoiler below.

>! I found the one on the top of the tower, and the one with the rocket launcer item. But I feel I looked everywhere. Does the third secret exist?

**Comment by RandomBrushes** [comment]
Hi, map author here.

Spoiler below.

When being in the last area (with the sword), the sky in one direction will have a small, blue disc, that's not moving like the rest of the skybox. If you jump from an elevated position, you can reach it. Have a look around and enjoy!

**Reply by Battojutsu** [reply]
> Hi, map author here.
> 
> Spoiler below.
> 
> When being in the last area (with the sword), the sky in one direction will have a small, blue disc, that's not moving like the rest of the skybox. If you jump from an elevated position, you can reach it. Have a look around and enjoy!

Oh, that's awesome. Thanks for the hint. I ended up getting to it.

Spoiler Below
I was able to rocket jump it, and if I timed it just right I could make it over there. I was almost certain it had something to do with the graves on the second island haha. I spent a long time trying to find something there.

By the way is there any significance to Bob? The guy in the first grave?

**Comment by RandomBrushes** [comment, marked as answer]
Glad you found the secret!

Bob is a short name that fits onto small, pixelated grave stones. In other, completely unrelated news, I grew up with the video game "The Secret of Monkey Island" - https://monkeyisland.fandom.com/wiki/Bob

Spoilers below.

A rocket jump is not really necessary - a normal jump from the elevated platforms does suffice.
